### PR TITLE
Fix alert-warn background

### PR DESCRIPTION
**Pull request by Alkarex**
Fix minor CSS regression from
https://github.com/FreshRSS/FreshRSS/pull/2990

### CODE REVIEW

File: p/themes/base-theme/template.css [hunk 1]
```diff
@@ -604,7 +608,7 @@ input[type="search"] {
 	table-layout: fixed;
 }
 
-#stream {
+#stream:not(.alert-warn) {
 	background: inherit;
```
Comment (Alkarex) [inline p/themes/base-theme/template.css:612]: This rule otherwise had priority in other themes overriding `.alert-warn  { background: XXX }`

**Comment by Alkarex** [comment]
Before the fix:
image

After the fix:
image

File: p/themes/base-theme/template.css [hunk 2]
```diff
@@ -384,6 +384,10 @@ a.btn {
 	width: 90%;
 }
 
+.alert-warn {
+	background: inherit;
+}
```
Comment (marienfressinaud) [inline p/themes/base-theme/template.css:389]: Why do we need these lines? The `#stream:not(.alert-warn)` seems to be enough, isn't it?

Comment (Alkarex) [reply p/themes/base-theme/template.css:389]: Yes indeed. It was just to make sure to have a background in all cases

**Pull request merged**